teamManager/teamManager.py

```diff
@@ -419,6 +421,18 @@ async def teams(self, ctx, *, franchise_tier_identifier: str):
 
         await ctx.send("No tier, franchise, prefix, or GM with name: {0}".format(franchise_tier_identifier))
 
+    @commands.command()
+    @commands.guild_only()
+    async def getTeam(self, ctx: commands.Context, user: discord.Member) -> None:
```

Comment:
I know this convention is used elsewhere, but we should get away from it.

`user: discord.Member` is what I'm taking issue with. We should probably rename the variable to `member` or `player`.

`discord.User` and `discord.Member` are two different data types. `Member` is a `User` with a `Guild` association. So naming the player variable `user` isn't an optimal alignment with the data type.

No functional issues, but I think updating these inconsistencies should be part of some of our future refactoring effort.

Reply:
I followed the same argument style as the functions called underneath in `get_current_team_name` and it `get_current_tier_role`.

If we want to change them, let me know.